features/Dynamic Cubemaps/Shaders/DynamicCubemaps/InferCubemapCS.hlsl

```diff
@@ -98,9 +90,9 @@ float3 Lin2sRGB(float3 color)
 	}
 
 #if defined(REFLECTIONS)
-	color.rgb = lerp(color.rgb, sRGB2Lin(ReflectionsTexture.SampleLevel(LinearSampler, uv, 0)), saturate(mipLevel * (1.0 / 10.0)));
+	color.rgb = lerp(color.rgb, sRGB2Lin(ReflectionsTexture.SampleLevel(LinearSampler, uv, 0)), saturate(mipLevel * (1.0 / 8.0)));
```

Comment:
why `(1.0 / 8.0)` instead of simply dividing?

Reply:
We can just divide, yeah.